### ISSUE TITLE
Binary16 Support

### ISSUE THREAD

**Issue by no-identd** [cvickery/IEEE-754#3]
I know the FAQ says there exist plans for it, but that was 6 years ago, so I figured I'd open a github issue... :)

**Comment by cvickery** [comment]
Good point. It's (obviously) been low priority for me, but having here is a good reminder that it's on the to-do list.
